ui/src/app/components/webcomponents/test-plan-platform-os-version-form.component.ts

```diff
@@ -152,7 +157,7 @@ export class TestPlanPlatformOsVersionFormComponent implements OnInit {
       this.browsers = res;
       if (!this.isPrivateGrid) {
         if (this.environmentFormGroup?.controls['browser']?.value)
-          this.platformBrowser = this.browsers.find(browser => browser.id == this.environmentFormGroup?.controls['browser']?.value);
+          this.platformBrowser = this.browsers.find(browser => browser.name.toLowerCase().includes(this.environmentFormGroup?.controls['browser']?.value.toLowerCase()) );
```

Comment:
what is the need of includes here?

Reply:
i think not need, we can check using ' == ' only